src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -57,13 +57,19 @@ public DarCollectionService(DarCollectionDAO darCollectionDAO, DatasetDAO datase
    * @return A PaginationResponse object
    */
   public PaginationResponse<DarCollection> getCollectionsWithFilters(PaginationToken token, User user) {
-
     List<DarCollection> unfilteredDars = darCollectionDAO.findDARCollectionsCreatedByUserId(user.getDacUserId());
+    if(unfilteredDars.isEmpty()) {
```

Comment:
nit: spacing. Here and below, see our style guide for whitespace
* https://github.com/DataBiosphere/consent/blob/develop/CONTRIBUTING.md
* https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace
```suggestion
    if (unfilteredDars.isEmpty()) {
```